pwa/src/components/objectEntities/ObjectEntityForm.tsx

```diff
@@ -237,7 +237,7 @@ export const ObjectEntityForm: React.FC<ObjectEntityFormProps> = ({ objectId, en
                               objectEntity.application !== null ? (
                                 <SelectInputComponent
                                   options={applications}
-                                  data={objectEntity.application.name}
+                                  data={objectEntity.applications.id}
```

Comment:
This entire file is never included anywhere, so I feel as if this change did nothing (and was not tested, I assume).